src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -35,6 +35,7 @@
         "symfony/proxy-manager-bridge": "Generate service proxies to lazy load them"
     },
     "conflict": {
+        "ext-psr": "<1.1",
```

Comment:
I wonder if this new constraint will not just prevent people to install next patch version of Symfony, but composer will install (or stay) on the current "buggy" version... And this won't fix the WTF of the current situation :-(

Reply:
> I wonder if this new constraint will not just prevent people to install next patch version of Symfony

No, it will prevent the installation of Symfony 5.3 if we merge before 5.3.0 is tagged. And if the installation of Symfony 5.3 is forced by the developer, Composer will output a nice explanation why this is not possible.

Reply:
indeed , si missed this PR targets 5.x